docs/application/native/guides/app-management/common-appcontrol.md

```diff
@@ -2239,6 +2239,83 @@ vpn_appcontrol_result_cb(app_control_h request, app_control_h reply, app_control
     app_control_get_extra_data(reply, APP_CONTROL_DATA_TEXT, &result_txt);
     dlog_print(DLOG_INFO, LOG_TAG, "Result: %s", result_txt);
 }
+```
+## Privacy setting guide popup
+
+The Privacy setting guide popup application control is supported only in wearable applications since Tizen 5.0.
+
+### Shows privacy related common guide popup
+
+To show common privacy setting guide popup, use the `APP_CONTROL_OPERATION_PRIVACY_SETTING_GUIDE` operation. This operation helps you to guide users to change privacy privilege settings. You need to check permissions to see if the user actually granted requested permissions after using this application control.
+
+**Figure: Privacy setting guide popup**
+
+![Guide popup for app launching](./media/common_appcontrol_privacy_setting_guide_open.png)![Guide popup for using privacy privileged function](./media/common_appcontrol_privacy_setting_guide_use.png)![Required privacy list shown in popup](./media/common_appcontrol_privacy_setting_guide_privacy.png)
+
+#### Operation
+
+- `http://tizen.org/appcontrol/operation/setting/guide_privacy_setting` (in `.c` files and manifest file)
+- `APP_CONTROL_OPERATION_PRIVACY_SETTING_GUIDE` (in `.c` files only)
+
+#### Extra Input
+
+| Key                     | Description                              | Note                   |
+|-------------------------|------------------------------------------|------------------------|
+| `APP_CONTROL_DATA_TYPE` | The guide message type. The available values are `open` (for application launching) and `use` (for using function).  This key must be passed as a string. | This key is mandatory. |
+| `APP_CONTROL_DATA_FEATURE` | The feature or function name to show. This key must be passed as a string. |   |
+| `APP_CONTROL_DATA_PRIVILEGES` | The required privacy privileges to show. This key must be passed as an array of string.| The required privacy privileges to show. This key must be passed as an array of string. |
```

Comment:
Modified note of 'APP_CONTROL_DATA_PRIVILEGES'. 

For description, I added "to show" because the app developer can select some of the privacy privileges. For example, A, B, and C are required privacy privileges of the application but the developer can decide to request user only A and B for now. I just added it to say "don't need to add all privacy privileges of the application". Should it be modified? Need your opinion.

Reply:
Ok. 
Then can we rephrase it to: 
The required privacy privileges that are selected.
OR
The required privacy privileges, which are displayed.
OR
The required privacy privileges that are requested.
OR
The required and requested privacy privileges.
OR
You can rephrase it better.But, we will avoid "to show". Need your opinion.

Reply:
You mean.. you're avoiding ending a sentence with "to OOO" form? If it's not, "to display" or "to request" would be OK.
Or I want to use 'The required privacy privileges that are requested.'.
Need your opinion.

Reply:
As a reader, when I read 'The required privacy privileges to show' or 'The required privacy privileges to display' confuses me and breaks the flow for me. Because I feel like something is to show or something is to get displayed.
If you use:
'The required privacy privileges that are requested.' Then your description " For example, A, B, and C are required privacy privileges of the application but the developer can decide to request user only A and B for now." goes well with this.

Hence, I feel we should stick to something like 'The required privacy privileges that are requested.'.
Need your opinion.

Reply:
Modified as 'The required privacy privileges that are requested.'. Thank you for kind explanation.

Reply:
Thank you for understanding and updating the changes.

Reply:
Hello. Reni-90, and yl33

I can see you had discussion about this "The required privacy privileges that are requested." 

However unfortunately at the first glance, I couldn't get the meaning of the sentence... As it sounds like "Asked privacy privileges asked.". The discussion history here barely made me understand, but the users out there can't read this discussion.

So Reni-90, I hope we get better sentence than 'The required privacy privileges that are requested.`. 
Would you please think about it more to clarify the meaning of "requested privacy privilege to an user, among required privileges"?
Or I think we can focus on the popup description itself so to change the sentence like "The  privacy privileges to be requested on popup", not comparing whole required privileges and requested privileges.

Regards